src/xpk/core/kueue.py

```diff
@@ -177,6 +212,20 @@ def install_kueue_on_cluster(args) -> int:
   Returns:
     0 if successful and 1 otherwise.
   """
+
+  err_code, kueue_version_installed = get_kueue_version(args)
+  if err_code == 0:
+    if Version(kueue_version_installed) <= Version('v0.8.1') and Version(
+        KUEUE_VERSION
+    ) >= Version('v0.9.1'):
```

Comment:
The deletion of those CRDs is listed as a manual upgrade step for Kueue v0.9.0 release so why the condition is not simply Version(kueue_version_installed) < Version('v0.9.0') and Version(
        KUEUE_VERSION
    ) >= Version('v0.9.0')